src/reducers/example.tsx

```diff
@@ -5,7 +5,7 @@ type TogglePayload = {};
 
 export const actions = {
   toggle: createAction('TOGGLE', (resolve) => {
-    return () => resolve({} as TogglePayload);
```

Comment:
I think we need an example of how to accomplish this pattern. I don't know if this is the best way but I'd say maybe keep this as is with a comment disabling the rule. If you do want to delete it, you can delete the type definition for `TogglePayload`, too (why didn't unused vars catch that? Hmm).

Reply:
removed everything